distribution/docs/src/main/jdocs/content/_plugins/event-processor-contents.adoc

```diff
@@ -0,0 +1,77 @@
+:type: plugin
+:status: published
+:title: Event Processor
+:link: _event_processor
+:plugintypes: postingest
+:summary: Creates, updates, and deletes subscriptions.
+
+The Event Processor provides an engine that creates, updates, and deletes subscriptions for event notification.
+These subscriptions optionally specify a filter criteria so that only events of interest to the subscriber are posted for notification.
+
+An internal subscription tracker monitors the OSGi registry, looking for subscriptions to be added (or deleted).
+When it detects a subscription being added, it informs the Event Processor, which sets up the subscription's filtering and is responsible for posting event notifications to the subscriber when events satisfying their criteria are met.
+
+The Standard Event Processor is an implementation of the Event Processor and provides the ability to create/delete subscriptions.
+Events are generated by the ${ddf-catalog}Framework as metacards are created/updated/deleted and the Standard Event Processor is called since it is also a Post-Ingest Plugin.
+The Standard Event Processor checks each event against each subscription's criteria.
+
+When an event matches a subscription's criteria the Standard Event Processor:
+
+* invokes each pre-delivery plugin on the metacard in the event.
+* invokes the Delivery Method's operation corresponding to the type of event being processed, e.g., created operation for the creation of a metacard.
```

Comment:
✏️  Since _Delivery Method_ is the name of an interface, should we use `DeliveryMethod` instead? We could also be specific and write that the `created()` method will be called in the example.